nifi-nar-bundles/nifi-framework-bundle/nifi-framework/nifi-client-dto/src/main/java/org/apache/nifi/web/api/entity/ProcessGroupUploadEntity.java

```diff
@@ -0,0 +1,111 @@
+/*
+ * Licensed to the Apache Software Foundation (ASF) under one or more
+ * contributor license agreements.  See the NOTICE file distributed with
+ * this work for additional information regarding copyright ownership.
+ * The ASF licenses this file to You under the Apache License, Version 2.0
+ * (the "License"); you may not use this file except in compliance with
+ * the License.  You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package org.apache.nifi.web.api.entity;
+
+import javax.xml.bind.annotation.XmlRootElement;
+import org.apache.nifi.registry.flow.VersionedFlowSnapshot;
+
+/**
+ * A serialized representation of this class can be placed in the entity body of a request or response to or from the API. This particular entity holds a reference to a ProcessGroupUploadDTO.
+ */
+@XmlRootElement(name = "processGroupUploadEntity")
+public class ProcessGroupUploadEntity extends Entity {
+    private String id;
+    private String groupName;
+    private Double positionX;
+    private Double positionY;
+    private String clientId;
+    private Boolean disconnectedNodeAcknowledged;
+    private VersionedFlowSnapshot versionedFlowSnapshot;
+
+    /**
+     * @return The group ID
+     */
+    public String getId() {
+        return id;
+    }
+
+    public void setId(String id) {
```

Comment:
If this is the ID of the process group, and not the ID of the Upload Entity, I would call this getGroupId/setGroupId.

Reply:
Changed to getGroupId and setGroupId.